apes.js

```diff
@@ -21,7 +21,7 @@ Usage:
 ,gram =`
 apeline ::= (lvalue) S* "=" S* (rvalue)*
 S       ::= [#x20#x09#x0A#x0D]+
-lvalue  ::= (sym) | (tvpair) | "[" S* (tvpair*) S* "]"
+lvalue  ::= (tvpair) | (sym) | "[" S* (tvpair S*) (","S* tvpair S*)* S* "]"
```

Comment:
@nmushegian I updated the lvalue grammar here to allow for `type sym`, `[type sym]` and `[type sym, type sym2, type sym3, ...]` lvalues in assignment expressions.